medicines/doc-index-updater/src/create_manager/mod.rs

```diff
@@ -50,8 +50,11 @@ impl ProcessRetrievalError for RetrievedMessage<CreateMessage> {
         e: anyhow::Error,
         state_manager: &StateManager,
     ) -> anyhow::Result<()> {
-        if e.to_string() == "Couldn't retrieve file: [-31] Failed opening remote file" {
-            tracing::info!("Updating state to errored and removing message");
+        if e.to_string() == "Couldn't retrieve file: [-31] Failed opening remote file".to_string() {
```

Comment:
why revert this?

Reply:
Ah, mistake, I didn’t see the subtle difference of the to_string() at the end